scss/base/generic.scss

```diff
@@ -21,3 +21,7 @@ button,
 [type="submit"] {
   -webkit-appearance: none;
 }
+
+label:hover {
+  cursor: pointer;
```

Comment:
Scoping this to `for` won't solve the issue with the current pattern, as there are no `for` attributes. The `label` is associated with the `input` because the `input` is a child of the `label`.  

You'd have to revise the selector like:
```css
label[for]:hover,
label input:hover { cursor: pointer; }
```
And then you'd still need to account for hovering over the `span`s within these `label`s w/out `for` attributes.

Reply:
I don’t see the value in adding the `label input:hover` selector, since that would make the _input_ get the pointer cursor, not the label itself. Using the `for` attribute as this PR suggests is the correct solution here, and I would add that the markup where the `input` is nested in the `label` should be modified to use `for`.

My suggestion here still stands: the pointer cursor should not be applied to `label` elements that do nothing for the user when clicked.

Reply:
Hi @agarzola,

I will re-explain for you.

As @BcRikko requested previously:
> If possible, I don't want IDs attached to `input`.

Being that `id`s are not required on an `input` if it is a child of a `label`, this is where your solution alone would not cover the present markup pattern and the desire expressed to not used `id` on inputs if not necessary.  

I'm not disagreeing with your opinion to include the `label[for]:hover` selector.  Though I tend to side with the [arguments made in this article regarding mouse cursors](https://medium.com/simple-human/buttons-shouldnt-have-a-hand-cursor-b11e99ca374b), it's more important to be consistent and ensure to not let corner cases slip through if the goal is to buck native functionality and use pointer cursors on all interactive elements.

I hope this helps you understand the value of my previous comment.

Reply:
Hey, @scottaohara. 👋 Thanks for expanding on your previous comment.

To clarify: I do see value in your comment (and this exchange!). 😄 I just don’t see the value in making the `input` itself get the pointer cursor, which is what adding the `label input:hover` selector would do.

We’re in agreement that labels shouldn’t get pointer cursors in the first place, as well as the assertion that —conventions not withstanding— consistency is paramount. And it’s consistency in the experience that I’m trying to address with my suggestion, because a `label` _without_ a `for` attribute and no nested `input` does nothing for the user when clicked, so why should the cursor change, implying that the label is interactive?

So again, while we’re in agreement that the most appropriate answer _should be_ “let’s remove pointer cursor from `label` elements altogether”, if one must be applied then I think the only way to be as consistent as possible is to avoid the cursor change when hovering over inert elements.

I hope that clarifies my position on this.

Reply:
You’re right. I missed that! 😄 Yes: the correct solution is to revert the work that gives labels a pointer cursor.